framework/include/reporters/ReporterContext.h

```diff
@@ -85,6 +85,9 @@ class ReporterContextBase : public libMesh::ParallelObject
   /// and include all the data including the old values.
   ///
   /// This method only outputs the current value within the JSONOutput object.
+  /// NOTE: nlohmann qualification is needed for argument json because std::vector specialization is
```

Comment:
I said "specialization" earlier but pedantically it's "overload"

Reply:
can I fix it in next PR? Follow up for capability w/ transfers